lizmap/lizmap-modules/localiz/controllers/default.classic.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+* @package   lizmap
+* @subpackage localiz
+* @author    your name
+* @copyright 2011 3liz
+* @link      http://3liz.com
+* @license    All rights reserved
+*/
+
+class defaultCtrl extends jController {
+    /**
+    *
+    */
+    function index() {
+        $rep = $this->getResponse('html');
+        return $rep;
+    }
+    
+    /**
+    *
+    */
+    function search() {
+        $localConfig = jApp::configPath('localconfig.ini.php');
+        $ini = new jIniFileModifier($localConfig);
+        
```

Comment:
First, you shouldn't use jIniFileModifier to only read an ini file : it is very slow compare to parse_ini_file().

Second, all parameters into localconfig.ini.php are merged at run time with other configuration files, so in fact, you can access to your parameters with `jApp::config()->localiz_plugin` which is an array (ex: jApp::config()->localiz_plugin['db_host'])